lib/movie/screens/movie_details_screen/movie_details_screen.dart

```diff
@@ -160,7 +166,12 @@ class _Poster extends StatelessWidget {
         borderRadius: BorderRadius.circular(_kRadius),
         image: DecorationImage(
           fit: BoxFit.cover,
-          image: NetworkImage(posterPath),
+          image: posterPath.isNotEmpty
+              ? NetworkImage(posterPath)
+              : Image.asset(
+                  'assets/images/logo2.png',
```

Comment:
1) rename this png file
2) in infrastructure folder create file inside witch class MovieImage and add this 'assets/images/logo2.png' into the class